src/components/Icon/Icon.stories.scss

```diff
@@ -1,7 +1,4 @@
-@import "~vanilla-framework/scss/settings";
```

Comment:
Removing unnecessary individual imports.
It's all included in full Vanilla import and there is no downside to import it all (no CSS produced until anything is included).